styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -342,7 +341,6 @@
   width: 250px;
 }
 
-
```

Comment:
Would you mind just dropping this file from the PR altogether?  Nothing ended up changing here, so I'd prefer just leaving the file untouched.